amida-tech/indaba-client
# Feature/inba 79 task option modal

#### What's this PR do?
Creates the modal whose appearance and values are mapped to the `project.ui` reducer state. Does not yet handle mapping to the project data itself.

#### Related JIRA tickets:
INBA-79, and soon INBA-128 for mapping to the reducer state of project or tasks.

#### How should this be manually tested?
Load `localhost:3000/project`
Click on a task's ellipsis
Observe the modal that appears, try playing with some of the fields.

#### Any background context you want to provide?

#### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4026454/27440574-330b0f48-5739-11e7-95df-3b689c592f1c.png)
